omod/src/main/webapp/pages/labs/labs.gsp

```diff
@@ -14,7 +14,7 @@
     <% if (context.hasPrivilege("App: labs")) { %>
         <a class="button app big" href='/${contextPath}/owa/labworkflow/index.html'>
             <i class="icon-beaker"></i>
-            ${ ui.message("pih.app.labtracking") }
```

Comment:
I'm not sure where this page is used, or if it still used, but this label seemed wrong.